examples/routing_example.ipynb

```diff
@@ -1058,9 +1355,10 @@
     "graph_routed_circuit = route(gp_ex_circ, ourense_device)\n",
     "noise_aware_routed_circuit = route(np_ex_circ, ourense_device)\n",
     "\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(line_routed_circuit)\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(graph_routed_circuit)\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(noise_aware_routed_circuit)\n",
+    "\n",
+    "for c in [line_routed_circuit, graph_routed_circuit, noise_aware_routed_circuit]:\n",
+    "    Transform.DecomposeBRIDGE().apply(c)\n",
+    "    Transform.DecomposeSWAPtoCX().apply(c)\n",
```

Comment:
No `RemoveRedundancies`?

Reply:
We could take the opportunity to illustrate the `SequencePass` construction (making a `SequencePass` once and then applying it to each circuit).

Reply:
`SequencePass ` along with `CompilationUnit` is introduced at the end of the notebook, so while I considered this I decided it was better to first introduce them together at the end. 

Regarding removing `RemoveRedundancies`, I thought that it was unnecessary and potentially overcomplicated that example at that point, so it was a deliberate decision.